package.json

```diff
@@ -111,7 +111,7 @@
     "ember-inflector": "^2.0.0"
   },
   "engines": {
-    "node": ">= 0.12.0"
```

Comment:
Yep, the changes are good. I was just commenting to make it clear to the other maintainers that Node 4 was essentially already required...